scripts/releases/update-gh-pages.sh

```diff
@@ -81,6 +81,5 @@ git commit -S -m "$commit_msg"
 
 if [ -n "$push_remote" ]; then
     echo "Pushing gh-pages to $push_remote"
-    git push "$push_remote" gh-pages
-
+    git push "$push_remote" HEAD:gh-pages --force
```

Comment:
We don't need '--force' here. 'git push' should work if there were no issues in previous steps